llvm/lib/Target/DirectX/DXIL.td

```diff
@@ -86,57 +106,85 @@ class dxil_op<string name, int code_id, dxil_class code_class, dxil_category op_
   let stats_group = _stats_group;
 }
 
-// The intrinsic which map directly to this dxil op.
+class dxil_op_min<string _name, int code_id, string _doc> : dxil_inst {
```

Comment:
dxil_op_min is confusing. Feels like only for the min operation.

Reply:
Agreed. Will change. Thanks!